spec/suites/map/MapSpec.js

```diff
@@ -1259,6 +1259,83 @@ describe("Map", function () {
 		});
 	});
 
+	describe("#flyToBounds", function () {
+
+		beforeEach(function () {
+			container.style.width = "800px";
+			container.style.height = "600px";
+			container.style.visibility = "hidden";
```

Comment:
Is it possible to use the default style? In `createContainer` width and height are set to 400